target/linux/x86/config-6.6

```diff
@@ -212,6 +218,8 @@ CONFIG_KEXEC=y
 CONFIG_KEXEC_CORE=y
 CONFIG_KEYBOARD_ATKBD=y
 CONFIG_KMAP_LOCAL=y
+# CONFIG_KVM_PROVE_MMU is not set
+# CONFIG_KVM_SMM is not set
```

Comment:
I'm running a qemu/kvm on an x86 openwrt host, and this breaks it.
`qemu-system-x86_64: System Management Mode not supported by this hypervisor.`

Please remove this line and add it to package/kernel/linux/modules/virt.mk (see my other comment)

Reply:
patch removed